### ISSUE TITLE
gdal not working with latest poppler builds

### ISSUE THREAD

**Issue by cpaulik** [conda-forge/libgdal-feedstock#25]
Since today a fresh gdal install breaks on my system because of new poppler versions.

```
miniconda2-4.3.30 ❯ conda create -n test-gdal -c conda-forge gdal
Fetching package metadata .............
Solving package specifications: .

Package plan for installation in environment /home/cpa/.pyenv/versions/miniconda2-4.3.30/envs/test-gdal:

The following NEW packages will be INSTALLED:

    ca-certificates: 2017.11.5-0           conda-forge
    cairo:           1.14.6-5              conda-forge
    certifi:         2017.11.5-py27_0      conda-forge
    curl:            7.55.1-0              conda-forge
    expat:           2.1.0-3               conda-forge
    fontconfig:      2.12.1-6              conda-forge
    freetype:        2.7-2                 conda-forge
    freexl:          1.0.4-0               conda-forge
    gdal:            2.2.2-py27_1          conda-forge
    geos:            3.6.2-1               conda-forge
    geotiff:         1.4.2-0               conda-forge
    gettext:         0.19.7-1              conda-forge
    giflib:          5.1.4-0               conda-forge
    glib:            2.51.4-0              conda-forge
    hdf4:            4.2.13-0              conda-forge
    hdf5:            1.10.1-1              conda-forge
    icu:             58.2-0                conda-forge
    intel-openmp:    2018.0.0-hc7b2577_8              
    jpeg:            9b-2                  conda-forge
    json-c:          0.12.1-0              conda-forge
    kealib:          1.4.7-4               conda-forge
    krb5:            1.14.2-0              conda-forge
    libdap4:         3.18.3-2              conda-forge
    libffi:          3.2.1-3               conda-forge
    libgdal:         2.2.2-7               conda-forge
    libgfortran:     3.0.0-1                          
    libiconv:        1.15-0                conda-forge
    libkml:          1.3.0-3               conda-forge
    libnetcdf:       4.4.1.1-10            conda-forge
    libpng:          1.6.28-2              conda-forge
    libpq:           9.6.3-0               conda-forge
    libspatialite:   4.3.0a-17             conda-forge
    libssh2:         1.8.0-2               conda-forge
    libtiff:         4.0.7-1               conda-forge
    libxml2:         2.9.5-2               conda-forge
    mkl:             2018.0.1-h19d6760_4              
    ncurses:         5.9-10                conda-forge
    numpy:           1.13.3-py27hbcc08e0_0            
    openjpeg:        2.1.2-3               conda-forge
    openssl:         1.0.2l-0              conda-forge
    pcre:            8.39-0                conda-forge
    pip:             9.0.1-py27_0          conda-forge
    pixman:          0.34.0-0              conda-forge
    poppler:         0.61.1-1              conda-forge
    poppler-data:    0.4.8-0               conda-forge
    proj4:           4.9.3-5               conda-forge
    python:          2.7.14-1              conda-forge
    readline:        6.2-0                 conda-forge
    setuptools:      36.7.2-py27_0         conda-forge
    sqlite:          3.13.0-1              conda-forge
    tk:              8.5.19-2              conda-forge
    util-linux:      2.21-0                           
    wheel:           0.30.0-py_1           conda-forge
    xerces-c:        3.1.4-3               conda-forge
    xz:              5.2.3-0               conda-forge
    zlib:            1.2.11-0              conda-forge

Proceed ([y]/n)? y

#
# To activate this environment, use:
# > source activate test-gdal
#
# To deactivate an active environment, use:
# > source deactivate
#


~/.pyenv/versions/miniconda2-4.3.30/conda-bld/mapnik_1511517460177/work master ⇣ 20s
miniconda2-4.3.30 ❯ pyenv shell miniconda2-4.3.30/envs/test-gdal 

~/.pyenv/versions/miniconda2-4.3.30/conda-bld/mapnik_1511517460177/work master ⇣
test-gdal ❯ python
Python 2.7.14 | packaged by conda-forge | (default, Nov  4 2017, 10:15:28) 
[GCC 4.8.2 20140120 (Red Hat 4.8.2-15)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import gdal
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/cpa/.pyenv/versions/miniconda2-4.3.30/envs/test-gdal/lib/python2.7/site-packages/gdal.py", line 2, in <module>
    from osgeo.gdal import deprecation_warn
  File "/home/cpa/.pyenv/versions/miniconda2-4.3.30/envs/test-gdal/lib/python2.7/site-packages/osgeo/__init__.py", line 21, in <module>
    _gdal = swig_import_helper()
  File "/home/cpa/.pyenv/versions/miniconda2-4.3.30/envs/test-gdal/lib/python2.7/site-packages/osgeo/__init__.py", line 17, in swig_import_helper
    _mod = imp.load_module('_gdal', fp, pathname, description)
ImportError: libpoppler.so.66: cannot open shared object file: No such file or directory
```

### Downgrading poppler fixes the problem

```
~/.pyenv/versions/miniconda2-4.3.30/conda-bld/mapnik_1511517460177/work master ⇣ 53s
test-gdal ❯ conda install 'poppler<0.61.1' gdal=2.2.2
Fetching package metadata .............
Solving package specifications: .

Package plan for installation in environment /home/cpa/.pyenv/versions/miniconda2-4.3.30/envs/test-gdal:

The following packages will be DOWNGRADED:

    poppler: 0.61.1-1 conda-forge --> 0.52.0-3 conda-forge

Proceed ([y]/n)? y


~/.pyenv/versions/miniconda2-4.3.30/conda-bld/mapnik_1511517460177/work master ⇣ 11s
test-gdal ❯ python
Python 2.7.14 | packaged by conda-forge | (default, Nov  4 2017, 10:15:28) 
[GCC 4.8.2 20140120 (Red Hat 4.8.2-15)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import gdal
>>> 
```

**Comment by ocefpaf** [comment]
Can you try it again? This should be fixed by #24

**Comment by cpaulik** [comment]
Now it seems even more broken since I also have to downgrade xerces-c

```
vds_s1 ❯ conda create -n test-gdal -c conda-forge gdal
Fetching package metadata .............
Solving package specifications: .

Package plan for installation in environment /home/cpa/.pyenv/versions/miniconda2-4.3.30/envs/test-gdal:

The following NEW packages will be INSTALLED:

    boost:           1.65.1-py27_0         conda-forge
    boost-cpp:       1.65.1-1              conda-forge
    bzip2:           1.0.6-1               conda-forge
    ca-certificates: 2017.11.5-0           conda-forge
    cairo:           1.14.6-5              conda-forge
    certifi:         2017.11.5-py27_0      conda-forge
    curl:            7.55.1-0              conda-forge
    expat:           2.1.0-3               conda-forge
    fontconfig:      2.12.1-6              conda-forge
    freetype:        2.7-2                 conda-forge
    freexl:          1.0.4-0               conda-forge
    gdal:            2.2.2-py27_1          conda-forge
    geos:            3.6.2-1               conda-forge
    geotiff:         1.4.2-0               conda-forge
    gettext:         0.19.7-1              conda-forge
    giflib:          5.1.4-0               conda-forge
    glib:            2.51.4-0              conda-forge
    hdf4:            4.2.13-0              conda-forge
    hdf5:            1.10.1-1              conda-forge
    icu:             58.2-0                conda-forge
    intel-openmp:    2018.0.0-hc7b2577_8              
    jpeg:            9b-2                  conda-forge
    json-c:          0.12.1-0              conda-forge
    kealib:          1.4.7-4               conda-forge
    krb5:            1.14.2-0              conda-forge
    libdap4:         3.18.3-2              conda-forge
    libffi:          3.2.1-3               conda-forge
    libgdal:         2.2.2-7               conda-forge
    libgfortran:     3.0.0-1                          
    libiconv:        1.15-0                conda-forge
    libkml:          1.3.0-4               conda-forge
    libnetcdf:       4.4.1.1-10            conda-forge
    libpng:          1.6.28-2              conda-forge
    libpq:           9.6.3-0               conda-forge
    libspatialite:   4.3.0a-18             conda-forge
    libssh2:         1.8.0-2               conda-forge
    libtiff:         4.0.7-1               conda-forge
    libxml2:         2.9.5-2               conda-forge
    mkl:             2018.0.1-h19d6760_4              
    ncurses:         5.9-10                conda-forge
    numpy:           1.13.3-py27hbcc08e0_0            
    openjpeg:        2.1.2-3               conda-forge
    openssl:         1.0.2l-0              conda-forge
    pcre:            8.39-0                conda-forge
    pip:             9.0.1-py27_0          conda-forge
    pixman:          0.34.0-1              conda-forge
    poppler:         0.61.1-1              conda-forge
    poppler-data:    0.4.8-0               conda-forge
    proj4:           4.9.3-5               conda-forge
    python:          2.7.14-1              conda-forge
    readline:        6.2-0                 conda-forge
    setuptools:      37.0.0-py27_0         conda-forge
    sqlite:          3.13.0-1              conda-forge
    tk:              8.5.19-2              conda-forge
    util-linux:      2.21-0                           
    wheel:           0.30.0-py_1           conda-forge
    xerces-c:        3.2.0-0               conda-forge
    xz:              5.2.3-0               conda-forge
    zlib:            1.2.11-0              conda-forge

Proceed ([y]/n)? y

pixman-0.34.0- 100% |#########################################################| Time: 0:00:01   3.62 MB/s
setuptools-37. 100% |#########################################################| Time: 0:00:00   4.66 MB/s
libkml-1.3.0-4 100% |#########################################################| Time: 0:00:01 518.48 kB/s
#
# To activate this environment, use:
# > source activate test-gdal
#
# To deactivate an active environment, use:
# > source deactivate
#


~/workspace/vds_s1/tests processing-improvements* 1m 16s
vds_s1 ❯ source activate test-gdal
pyenv-virtualenv: activate miniconda2-4.3.30/envs/test-gdal

~/workspace/vds_s1/tests processing-improvements*
test-gdal ❯ gdalinfo
gdalinfo: error while loading shared libraries: libpoppler.so.66: cannot open shared object file: No such file or directory

~/workspace/vds_s1/tests processing-improvements*
test-gdal ❯ conda install 'poppler<0.61.1' gdal=2.2.2
Fetching package metadata .............
Solving package specifications: .

Package plan for installation in environment /home/cpa/.pyenv/versions/miniconda2-4.3.30/envs/test-gdal:

The following packages will be DOWNGRADED:

    poppler: 0.61.1-1 conda-forge --> 0.52.0-3 conda-forge

Proceed ([y]/n)? y


~/workspace/vds_s1/tests processing-improvements* 11s
test-gdal ❯ gdalinfo                                 
gdalinfo: error while loading shared libraries: libxerces-c-3.1.so: cannot open shared object file: No such file or directory

~/workspace/vds_s1/tests processing-improvements*
test-gdal ❯ conda install 'xerces-c=3.1'
Fetching package metadata .............
Solving package specifications: .

Package plan for installation in environment /home/cpa/.pyenv/versions/miniconda2-4.3.30/envs/test-gdal:

The following packages will be DOWNGRADED:

    xerces-c: 3.2.0-0 conda-forge --> 3.1.4-3 conda-forge

Proceed ([y]/n)? y


~/workspace/vds_s1/tests processing-improvements* 11s
test-gdal ❯ gdalinfo
Usage: gdalinfo [--help-general] [-json] [-mm] [-stats] [-hist] [-nogcp] [-nomd]
                [-norat] [-noct] [-nofl] [-checksum] [-proj4]
                [-listmdd] [-mdd domain|`all`]*
                [-sd subdataset] [-oo NAME=VALUE]* datasetname

FAILURE: No datasource specified.

~/workspace/vds_s1/tests processing-improvements*
test-gdal ❯ 

```
